### ISSUE TITLE
Offline tiles are only shown on the second execution of app

### ISSUE THREAD

**Issue by lisandrorueckert** [osmdroid/osmdroid#1507]
## Issue Type

[ ] Question
[X] Bug
[ ] Improvement
[ ] Build system related
[ ] Performance
[ ] Documentation

## Description and/or steps/code to reproduce the problem

After some changes to adjust my app to be in compliance with google play, I noted that the offline tiles only are shown on the second execution of App. On the first execution, I can see into the logs the lines below.  On the subsequent execution, while the App is still on the recent app list, the offline tiles are loaded OK.
The log bellow is generated during the setContentView execution. 

2020-03-02 10:02:44.719 31857-31857/com.volare.atenastreet D/StorageUtils: /proc/mounts
2020-03-02 10:02:44.720 31857-31857/com.volare.atenastreet D/StorageUtils: rootfs / rootfs ro,seclabel,relatime 0 0
2020-03-02 10:02:44.720 31857-31857/com.volare.atenastreet D/StorageUtils: tmpfs /dev tmpfs rw,seclabel,nosuid,relatime,mode=755 0 0
2020-03-02 10:02:44.720 31857-31857/com.volare.atenastreet D/StorageUtils: devpts /dev/pts devpts rw,seclabel,relatime,mode=600 0 0
2020-03-02 10:02:44.721 31857-31857/com.volare.atenastreet D/StorageUtils: none /dev/cpuctl cgroup rw,relatime,cpu 0 0
2020-03-02 10:02:44.721 31857-31857/com.volare.atenastreet D/StorageUtils: proc /proc proc rw,relatime,gid=3009,hidepid=2 0 0
2020-03-02 10:02:44.721 31857-31857/com.volare.atenastreet D/StorageUtils: sysfs /sys sysfs rw,seclabel,relatime 0 0
2020-03-02 10:02:44.721 31857-31857/com.volare.atenastreet D/StorageUtils: selinuxfs /sys/fs/selinux selinuxfs rw,relatime 0 0
2020-03-02 10:02:44.721 31857-31857/com.volare.atenastreet D/StorageUtils: debugfs /sys/kernel/debug debugfs rw,seclabel,relatime 0 0
2020-03-02 10:02:44.722 31857-31857/com.volare.atenastreet D/StorageUtils: none /acct cgroup rw,relatime,cpuacct 0 0
2020-03-02 10:02:44.722 31857-31857/com.volare.atenastreet D/StorageUtils: tmpfs /mnt tmpfs rw,seclabel,relatime,mode=755,gid=1000 0 0
2020-03-02 10:02:44.722 31857-31857/com.volare.atenastreet D/StorageUtils: /dev/fuse /mnt/runtime/default/emulated fuse rw,nosuid,nodev,noexec,noatime,user_id=1023,group_id=1023,default_permissions,allow_other 0 0
2020-03-02 10:02:44.723 31857-31857/com.volare.atenastreet D/StorageUtils: /dev/fuse /mnt/runtime/read/emulated fuse rw,nosuid,nodev,noexec,noatime,user_id=1023,group_id=1023,default_permissions,allow_other 0 0
2020-03-02 10:02:44.724 31857-31857/com.volare.atenastreet D/StorageUtils: /dev/fuse /mnt/runtime/write/emulated fuse rw,nosuid,nodev,noexec,noatime,user_id=1023,group_id=1023,default_permissions,allow_other 0 0
2020-03-02 10:02:44.724 31857-31857/com.volare.atenastreet D/StorageUtils: /dev/block/vold/public:253,48 /mnt/media_rw/1CEE-2917 vfat rw,dirsync,nosuid,nodev,noexec,relatime,uid=1023,gid=1023,fmask=0007,dmask=0007,allow_utime=0020,codepage=437,iocharset=iso8859-1,shortname=mixed,utf8,errors=remount-ro 0 0
2020-03-02 10:02:44.727 31857-31857/com.volare.atenastreet D/StorageUtils: /dev/fuse /mnt/runtime/default/1CEE-2917 fuse rw,nosuid,nodev,noexec,noatime,user_id=1023,group_id=1023,default_permissions,allow_other 0 0
2020-03-02 10:02:44.727 31857-31857/com.volare.atenastreet D/StorageUtils: /dev/fuse /mnt/runtime/read/1CEE-2917 fuse rw,nosuid,nodev,noexec,noatime,user_id=1023,group_id=1023,default_permissions,allow_other 0 0
2020-03-02 10:02:44.728 31857-31857/com.volare.atenastreet D/StorageUtils: /dev/fuse /mnt/runtime/write/1CEE-2917 fuse rw,nosuid,nodev,noexec,noatime,user_id=1023,group_id=1023,default_permissions,allow_other 0 0
2020-03-02 10:02:44.728 31857-31857/com.volare.atenastreet D/StorageUtils: none /config configfs rw,relatime 0 0
2020-03-02 10:02:44.729 31857-31857/com.volare.atenastreet D/StorageUtils: /dev/block/vda /system ext4 ro,seclabel,relatime,data=ordered 0 0
2020-03-02 10:02:44.729 31857-31857/com.volare.atenastreet D/StorageUtils: /dev/block/vdb /cache ext4 rw,seclabel,nosuid,nodev,noatime,errors=panic,data=ordered 0 0
2020-03-02 10:02:44.729 31857-31857/com.volare.atenastreet D/StorageUtils: /dev/block/vdc /data ext4 rw,seclabel,nosuid,nodev,noatime,errors=panic,data=ordered 0 0
2020-03-02 10:02:44.729 31857-31857/com.volare.atenastreet D/StorageUtils: tmpfs /storage tmpfs rw,seclabel,relatime,mode=755,gid=1000 0 0
2020-03-02 10:02:44.729 31857-31857/com.volare.atenastreet D/StorageUtils: /dev/fuse /storage/emulated fuse rw,nosuid,nodev,noexec,noatime,user_id=1023,group_id=1023,default_permissions,allow_other 0 0
2020-03-02 10:02:44.729 31857-31857/com.volare.atenastreet D/StorageUtils: /dev/fuse /storage/1CEE-2917 fuse rw,nosuid,nodev,noexec,noatime,user_id=1023,group_id=1023,default_permissions,allow_other 0 0
2020-03-02 10:02:44.730 31857-31857/com.volare.atenastreet D/StorageUtils: tmpfs /storage/self tmpfs rw,seclabel,relatime,mode=755,gid=1000 0 0
2020-03-02 10:02:44.733 31857-31857/com.volare.atenastreet I/StorageUtils: /sdcard is writable
2020-03-02 10:02:44.741 31857-31857/com.volare.atenastreet I/StorageUtils: /storage/emulated/0 is writable
2020-03-02 10:02:44.783 31857-31857/com.volare.atenastreet I/StorageUtils: /mnt/sdcard is writable
2020-03-02 10:02:44.797 31857-31857/com.volare.atenastreet I/StorageUtils: /storage/emulated/0 is writable
2020-03-02 10:02:44.802 31857-31857/com.volare.atenastreet I/StorageUtils: /sdcard is writable
2020-03-02 10:02:44.806 31857-31857/com.volare.atenastreet I/StorageUtils: /mnt/sdcard is writable

## Environment

I have tried this behaviour using the emulators of android studio using Android 7.1 and with my ASUS smartphone with android 8

### Version of osmdroid the issue relates to:

6.1.4

**Comment by monsieurtanuki** [comment]
I cannot think of something specific that would cause that behavior.
What if you use the demo app: do you reproduce this bug?

**Comment by lisandrorueckert** [comment]
I think I found the problem but I don't know how to fix it.  When the app starts for the first time, the mapview will look for offline tiles on /osmdroid folder. On my oncreate method I'm changing this location to another folder. But this change only will be effective on the following executions. 
 I found this because, althought I was changing the folder using the method Configuration.getInstance().setOsmdroidBasePath("path-to-somewhere"),  the folder  /osmdroid were still being created. Then I put my tiles.zip there and the map was shown on the first execution.
Maybe there is another method that I need to call right after Configuration.getInstance().setOsmdroidBasePath to force the use of the new folder on current execution.
